woocommerce/woocommerce-ios
# Shipping Labels: Remove all feature flags

Part of #6269 


### Description

This PR aims to clean up the feature flags for the Shipping Labels project. We're currently having 5 flags for this project alone - considering that the last milestone has been released since October 2021, it's time to remove them.


### Testing instructions

#### Test shipping label creation eligibility
1. Make sure that your test store has installed and configured WCShip plugin.
2. Create an order on your test store.
3. On the app, navigate to Orders > select the created order.
4. On the Order Details screen, notice that the Create Shipping Label button is present. If you placed the order with cash-on-delivery payment method and set up WCPay in your store, make sure to collect payment first. After the payment is completed, the Create Shipping Label button should be present.

#### Test International shipping label
- Repeat steps 1-4 in the shipping label eligibility test. This time, make sure that your test order has the shipping address of a non-US country. Then tap Create Shipping Labels button to start the creation flow.
- Ship From: country can be updated to US and its USPS-supported territories.
- Ship To: country can be updated to any country other than the US. State of country is only required if the selected country has a list of states.
- Phone number is required for both Ship From and Ship To if customs form is required (order is international or is shipment between US and one of its military states).
- Customs row is present if customs form is required.
- Tapping on Customs row should navigate to Customs form. 
- Customs form requires all fields and items to be validated.
- Purchasing international shipping label with DHL carrier should result in a separate customs invoice to be printed. Customs invoice printing screen should be displayed after printing the shipping label. 
- There should be option for printing customs form from Order Details screen if a separate customs invoice is available. 

#### Test creating custom shipping label package
1. Open an order detail that is eligible for a shipping label.
2. Tap the button "Create Shipping Label.”
3. Confirm "Ship From" and "Ship To.”
4. In the package details section, tap on “Package Selected.”
5. On the Add New Package screen, create a new custom package (on the Custom Package tab) or select a service package (on the Service Package tab).
6. Tap the Done button and confirm your new package is selected on the "Package Selected" screen.
7. Tap the Done button and confirm your new package is listed in the package details section of the shipping label form.
8. Complete your shipping label purchase.
9. Repeat the steps above with another order. Now that you have existing packages on your store, after step 5 you should see a list of the packages already activated on your store with a "Create new package" button at the bottom of the screen. Tap that button to get to the Add New Package screen. After you add a package, confirm the new package is included (and selected) in the package list.

#### Test Payment method update flow
1. Make sure that your test store has installed WCShip plugin and payment methods added in the "Shipping Labels" section in WPAdmin > WooCommerce > Settings > Shipping > WooCommerce Shipping.
2. Open an order eligible for shipping label creation.
3. Click on Create shipping label.
4. Pass all steps until you reach the payment step.
5. Click on edit.
6. You should see the "Add another credit card" button and you should be able to press it for adding a new payment method.

#### Test multi-package support
- Install & activate the WooCommerce Shipping & Tax plugin.
- Add at least two new packages.
- Create more than one shipping label for the same order.
- Create more than one shipping label for the same order for an international address.

---
